pkg/storer/reserve.go

```diff
@@ -276,21 +281,29 @@ func (db *DB) removeChunk(ctx context.Context, address swarm.Address, batchID []
 
 func (db *DB) reserveCleanup(ctx context.Context) error {
 	dur := captureDuration(time.Now())
+	count := 0
 	defer func() {
 		db.metrics.MethodCallsDuration.WithLabelValues("reserve", "cleanup").Observe(dur())
+		db.metrics.ReserveSize.Set(float64(db.reserve.Size()))
+		db.logger.Info("cleanup finished", "total", count, "duration", dur())
```

Comment:
Would "removed" be better than "total" the latter of which might imply that it's the total size of the reserve?